### ISSUE TITLE
Failed OTB Application Compilation

### ISSUE THREAD

**Issue by jinxiangbo** [CNES/cars#18]
When I'm building CARS internal OTB remote modules, the final step: make
Desktop/cars/cars/otb_remote_module/app/otbConvertSensorToGeoPointFast.cxx:26:10: fatal error: otbForwardSensorModel.h: No such file or directory
   26 | #include "otbForwardSensorModel.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [app/CMakeFiles/otbapp_ConvertSensorToGeoPointFast.dir/build.make:76: app/CMakeFiles/otbapp_ConvertSensorToGeoPointFast.dir/otbConvertSensorToGeoPointFast.cxx.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:155: app/CMakeFiles/otbapp_ConvertSensorToGeoPointFast.dir/all] Error 2
make: *** [Makefile:136: all] Error 2

**Comment by dyoussef** [comment]
Hello [jinxiangbo](https://github.com/jinxiangbo),

Is the Orfeo Toolbox correctly installed on your computer?
It's also possible (and easier) to install cars without using remote modules: https://cars.readthedocs.io/en/stable/install.html

Regards, 
David

**Comment by jinxiangbo** [comment]
Dear Youssef,
Thank you for taking the time to reply. The issue has been solved. since the OTB updated to OTB 8.0.0, some files have gone. Therefore, the makefile has been invalid. So I turned to use the OTB 7.4.1, and the installation became smooth. And I know there are some convenient approaches to install. But if I want to deeply understand the project, I'd better compile the source code. Thank you once again for your contribution to the advancement of the industry through open-source code.
Best regards，
Jin
